### ISSUE TITLE
Error thrown on success #ST3

### ISSUE THREAD

**Issue by adamgoose** [gnarula/sublime-laravelgenerator#26]
When running the generate:model command in ST3, I get the error "Oh snap, failed to create model." All the meanwhile, the following file was successfully created:

```
<?php

class Product extends Eloquent {
    protected $guarded = array();

    public static $rules = array();
}
```

**Comment by bvangraafeiland** [comment]
Yeah this is because of a slight misplacement with slashes, see #25 for a fix (hasn't been pulled, so you'll have to edit the file yourself accordingly, but it's just 2 lines).
